gitlens/gitlens-faq.md

```diff
@@ -0,0 +1,52 @@
+---
+
+title: GitLens FAQ
+description: GitLens FAQ
+taxonomy:
+    category: gitlens
+
+---
+
+The answers to your important frequently asked questions.
+
+***
+
+### How do I sign in or out of my GitKraken GitLens+ account?
+
+To sign into your GitKraken GitLens+ account, click the GitLens icon in the left sidebar and select `Sign In`.
```

Comment:
Shouldn't this all be generic to just "GitLens account"? Free users and trial users also sign in/out this way. Or are we thinking because this is on the FAQ this is where paid users may come to find out how to utilize their paid license? CC @Axxzilla

Reply:
It should match whatever we are using on the pricing page and in-app. If we are no longer using GitLens+ we should make sure to also update the other pages, such as https://help.gitkraken.com/gitlens/gitlens-plus/. We should also confirm with Marketing and PMM that this is the case https://www.gitkraken.com/gitlens/pricing

Reply:
An account isn't needed for "GitLens Free", but I guess there are users who have an account without a paid plan. Trial users are trying out "GitLens Pro".

I'm ok with "GitLens account". Mainly want to make sure we're distinguishing between GitLens+ features and GitLens plans, since GitLens+ features can be used without an account for public and local repos.